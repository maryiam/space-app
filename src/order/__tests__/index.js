import React from 'react';
import 'jest-styled-components';
import renderer from 'react-test-renderer';
import { OrderRecap } from '../';

describe('order/', () => {
  describe('#render', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
        <OrderRecap
          order={{
            destination: {
              id: 0,
              name: 'destination 1',
              currencies: [0],
              price: 100
            },
            payment: { id: 0, name: 'payment 1', rate: 1.2, symbol: '$' }
          }}
          actions={{
            startTrip: jest.fn()
          }}
        />
      );
      expect(tree.toJSON()).toMatchSnapshot();
    });
  });
});
