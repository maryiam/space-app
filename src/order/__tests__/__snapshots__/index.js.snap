// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`order/ #render renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 {
  color: #fff;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 2rem;
}

.c2 {
  margin-left: 0.5rem;
  font-size: 1.5em;
  color: #ffe818;
}

.c3 {
  background: transparent;
  border: 2px solid #ffe818;
  padding: 1rem 2rem;
  font-size: 2em;
  text-transform: uppercase;
  color: #ffe818;
  font-family: 'Passion One',sans-serif;
}

<section
  className="c0"
>
  <div
    className="c1"
  >
    Price:
     
    <span
      className="c2"
    >
      120
       
      $
    </span>
  </div>
  <button
    className="c3"
    onClick={[Function]}
  >
    let's go
  </button>
</section>
`;
